### PR TITLE
OX-4-implement-auditing || Task is done

### PR DESCRIPTION
**Pull request by Jamshidx8ja**

### CODE REVIEW

**Comment by escobar0007** [comment]
Done

File: pom.xml [hunk 1]
```diff
@@ -34,6 +34,7 @@
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
 
+
```
Comment (byte-11) [inline pom.xml:37]: Remove empty line breaks

File: src/main/java/uz/pdp/olx/configuration/auditing/AuditingConfig.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import uz.pdp.olx.enitiy.User;
+import uz.pdp.olx.repository.UserRepository;
+import uz.pdp.olx.service.UserService;
+
+import java.util.Optional;
+
+@Component
+@RequiredArgsConstructor
+public class AuditingConfig implements AuditorAware<User> {
+//Userni Security Authenticationda olish kerak
```
Comment (byte-11) [inline src/main/java/uz/pdp/olx/configuration/auditing/AuditingConfig.java:16]: Leave Todo comment if it is required

File: src/main/java/uz/pdp/olx/enitiy/User.java [hunk 3]
```diff
@@ -25,17 +26,17 @@
...
 
     @Column(unique = true)
     private String username;
 
-    @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "id")
-    private UpdateDate updateDate;
+//    @OneToOne(fetch = FetchType.LAZY)
+//    @JoinColumn(name = "id")
+//    private UpdateDate updateDate;
```
Comment (byte-11) [inline src/main/java/uz/pdp/olx/enitiy/User.java:39]: Remove commented , unused annotation @Jamshidx8ja

File: src/main/resources/application.yml
Comment (byte-11) [inline src/main/resources/application.yml:1]: Do not push application.yml file changes @Jamshidx8ja  @escobar0007  @Sardor0305  @abdumalik23102003

**Review by byte-11 — COMMENTED** [review]
Viewed changes to pull request

**Pull request merged**